src/main/java/ro/msg/learning/shop/domain/primarykeys/OrderDetailKey.java

```diff
@@ -0,0 +1,13 @@
+package ro.msg.learning.shop.domain.primarykeys;
```

Comment:
Cool that you created a separate package for keys!

I suggest moving it under `domain` as we can say that the keys are part of the domain.